packages/synapse-react-client/src/components/EntityFinder/EntityFinder.tsx

```diff
@@ -188,114 +180,124 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
-            {searchActive ? (
-              <Button
-                variant="outlined"
-                color="primary"
-                onClick={() => {
-                  setSearchActive(false)
-                  setSearchInput('')
-                  setSearchTerms(undefined)
-                }}
-                startIcon={<ArrowBackOutlinedIcon />}
-                sx={{ flexShrink: 0, mr: 1 }}
-              >
-                Back to Browse
-              </Button>
-            ) : (
-              <Button
-                variant="contained"
-                color="primary"
-                className="EntityFinder__Search__SearchButton"
-                onClick={() => {
-                  setSearchActive(true)
-                  searchInputRef.current!.focus()
-                }}
-                startIcon={<SearchIcon />}
-                sx={{ flexShrink: 0 }}
-              >
-                {searchButtonText}
-              </Button>
-            )}
-            <TextField
-              className="EntityFinder__Search__Input"
-              aria-hidden={!searchActive}
-              data-active={searchActive}
-              inputRef={searchInputRef}
-              placeholder="Search by name, Wiki contents, or Synapse ID"
-              value={searchInput}
-              onChange={event => {
-                setSearchInput(event.target.value)
-              }}
-              onKeyDown={(event: any) => {
-                if (event.key === 'Enter') {
-                  if (event.target.value === '') {
-                    setSearchTerms(undefined)
-                  } else {
-                    setSearchTerms(event.target.value.trim().split(' '))
-                  }
+          <Tabs
+            value={currentTab}
+            onChange={handleTabChange}
+            sx={{ flex: 'none' }}
+          >
+            <Tab value={EntityFinderTab.BROWSE} label={'Browse'}></Tab>
+            <Tab
+              value={EntityFinderTab.SELECTED}
+              label={
+                <Stack direction={'row'} gap={0.5} alignItems={'center'}>
+                  <span>Selected</span>
+                  <Chip
+                    size={'small'}
+                    label={selectedEntities.size.toLocaleString()}
+                    sx={{
+                      backgroundColor: 'tertiary.500',
+                      color: 'grey.900',
+                      height: '21px',
+                    }}
+                  ></Chip>
+                </Stack>
+              }
+            ></Tab>
+          </Tabs>
+          <TextField
+            size={'small'}
+            placeholder="Search by name, wiki content, or SynID"
+            value={searchInput}
+            onChange={event => {
+              setSearchInput(event.target.value)
+            }}
+            onKeyDown={(event: any) => {
+              if (event.key === 'Enter') {
+                const trimmedInput = event.target.value.trim()
+                setSearchActive(!!trimmedInput)
```

Comment:
@nickgros : On search, I think we should change to the Browse tab.  When I'm on the Selected tab and do a search, the search results are not shown (I need to know to switch tabs to see the results)

Reply:
That's a good idea--in general this is weird so I will probably follow-up with design about this.